tutorial/markdown/java/spring-data/spring-data.md

```diff
@@ -1,56 +1,25 @@
 ---
 # frontmatter
 path: "/tutorial-quickstart-java-springdata"
-title: Couchbase With Spring-Boot and Spring Data
-short_title: Spring Data
+title: Quickstart in Couchbase with Java and Spring Data
+short_title: Java and Spring Data
 description:
-  - Build a REST API with Couchbase and Spring Data
-  - Learn how to configure the Couchbase SDK
-content_type: tutorial
+  - Learn how to configure Spring Data with Couchbase
+  - Explore key-based operations and SQL++ querying using Spring Data Couchbase repositories
```

Comment:
Same as above

Reply:
now adapted from python and golang tutorials